### ISSUE TITLE
Panels Programmatically

### ISSUE THREAD

**Issue by mmdock** [antoniocasero/Panels#13]
Is there a way to build Panels without the need for Storyboard?

**Comment by antoniocasero** [comment]
Of course, as long your VC conforms the protocol Panelable, you can inject it to the method 'show'.

**Comment by antoniocasero** [comment]
but I'm working to make Panelable nicer :)

**Comment by Savyalov** [comment]
"Trying to instantiate something that does not conform Panelable :("

import UIKit
import AVFoundation
import Panels


class ViewController12: UIViewController, Panelable {
    @IBOutlet var headerHeight: NSLayoutConstraint!
    @IBOutlet var headerPanel: UIView!
}
 My storyboard name is "Main". Help me, I do not understand what the problem is. Thank!

**Comment by antoniocasero** [comment]
Hi @Savyalov, sorry for the late response. Are you still having problems? There is an extension included in the framework to instantiate Panels from the storyboard, maybe it helps.

**Comment by Savyalov** [comment]
Thank! Already figured out.

**Comment by erniela** [comment]
> Thank! Already figured out.

What was your solution? I have the same problem doing programmatically.  Please help.

**Comment by Savyalov** [comment]
Good morning! I just deleted and installed again Panelable. Nothing else helped.

**Comment by shchoi82-robotcode** [comment]
Hi, 
I'm trying to implement without storyboard.
Its my code.. what is the problem?

**let panelManager = Panels(target: self)
let panelConfiguration = PanelConfiguration(size: .oneThird)
panelManager.show(panel: self, config: panelConfiguration, view: myView)**



myVIew is just existing UIView
self is UIController & Panelable

runtime error  
EXC_BAD_ACCESS
Panels.swift: 38 Line
parentViewController?.addContainer(container: panel)
